tests/migration/migration.go

```diff
@@ -3533,6 +3533,30 @@ func libvirtDomainIsPersistent(virtClient kubecli.KubevirtClient, vmi *v1.Virtua
 	return strings.Contains(stdout, vmi.Namespace+"_"+vmi.Name), nil
 }
 
+func libvirtDomainIsPaused(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (bool, error) {
+	namespace := testsuite.GetTestNamespace(vmi)
+	vmi, err := virtClient.VirtualMachineInstance(namespace).Get(context.Background(), vmi.Name, &metav1.GetOptions{})
+	if err != nil {
+		return false, err
+	}
+
+	vmiPod, err := libpod.GetRunningPodByLabel(virtClient, string(vmi.GetUID()), v1.CreatedByLabel, namespace, vmi.Status.NodeName)
+	if err != nil {
+		return false, err
+	}
+
+	stdout, stderr, err := exec.ExecuteCommandOnPodWithResults(
+		virtClient,
+		vmiPod,
+		tests.GetComputeContainerOfPod(vmiPod).Name,
```

Comment:
@codingben Can you please check this function call `GetComputeContainerOfPod`? I think this function was removed as part of the recent rebase of yours

Reply:
Yeah it was removed. Thanks for catching it, it was rebased successfully and I didn't notice error messages.

Reply:
It should be: `libpod.LookupComputeContainer`.